apps/right-to-rent-check/index.js

```diff
@@ -137,6 +139,9 @@ module.exports = {
     '/confirm': {
       next: '/complete'
     },
+    '/exit-dont-need-check': {
```

Comment:
I would recommend calling this `/no-check-required`, but check that with @Nadiahuq and @karypun